packages/client/hmi-client/src/components/Slider.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<div
+		:class="`slider ${isOpen ? 'open' : 'closed'}`"
+		:style="{ width: isOpen ? contentWidth : tabWidth }"
+	>
+		<div class="slider-content-container" :style="{ width: contentWidth }">
+			<div class="slider-content" :style="sidePanelContentStyle">
+				<slot name="content"></slot>
+			</div>
+		</div>
+		<div class="slider-tab" :style="sidePanelTabStyle">
+			<slot name="tab"></slot>
+		</div>
+	</div>
```

Comment:
So many divs... you want my eyes to bleed 😭 
Could we use `<main>`, `<header>` etc,